csharp/beetlex/web.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BeetleX.FastHttpApi" Version="1.9.5" />
+    <PackageReference Include="BeetleX.FastHttpApi.Hosting" Version="1.3.6" />
```

Comment:
could this work ?
```suggestion
    <PackageReference Include="BeetleX.FastHttpApi" Version="1.9.*" />
    <PackageReference Include="BeetleX.FastHttpApi.Hosting" Version="**" />
```

Reply:
cannot

Reply:
yes this ? the idea is to have a more relax constraint (on patch level) + having PR from `renovatebot` (which take care of update)